boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -71,6 +71,7 @@
 		sw1 = &button1;
 		sw2 = &button2;
 		sw3 = &button3;
+		config-shell-serial = &uart0;
```

Comment:
Shouldn't this be in the `chosen` section, just like `zephyr,console = &uart0;` ?

Reply:
I am not agains this.
Can someone point pros and cons for both solutions?

Reply:
@erwango @galak any thoughts on this?